crates/bevy_ecs/src/schedule/label.rs

```diff
@@ -13,7 +13,7 @@ pub trait Label<T>: DynHash + DynClone<T> + Send + Sync + 'static {
 
 impl<M: 'static> Debug for dyn Label<M> {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
-        f.write_str(&self.name())
+        f.write_str(&format!("\"{}\"", &self.name()))
```

Comment:
```suggestion
        write!(f, "{:?}", self.name())
```

This will also put it between `"`, but avoids an allocation and will escapes `"` and special characters.